app/sequencescape_excel/sequencescape_excel/helpers/worksheet.rb

```diff
@@ -11,7 +11,8 @@ module Worksheet
           border: {
             style: :thin,
             color: '00'
-          }
```

Comment:
Makes the column 'text' rather than 'General' preventing Excel from being 'helpful' and ensuring the validations can function properly.